bootstrap.sh

```diff
@@ -0,0 +1,39 @@
+# Set the path for the file we are going to generate
+OUTPUT="$(pwd)/setup.txt"
```

Comment:
Instead of the `pwd` - could we use the bin folder (where the Oni2 executable ends up)? We can get that with `$cur__bin`.

Reply:
@bryphe I'm not sure where this variable exists? it errors when I add it to the script is it coming from esy? do I need to run a different esy command?

Reply:
Woops it does exist was accessing it incorrectly